### DISCUSSION TITLE
btc_vehicles folder issue

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1568] [Question and answer] [answered]
hi Vdauphin.

I have an issue when i transfer all vehicles i want to use in "btc_vehicles" folder in Eden but when i play, vehicles won't appear.
And when i transfer them in the "empty" folder, vehicles appear.
Can you help me ?
Thanks

**Comment by Vdauphin** [comment]
Hello,
You should delete your database
Cheers

**Reply by SOF-Ice** [reply]
Which folders are concerned ? core/fnc/db ? core/fnc/data ?

**Reply by Vdauphin** [reply, marked as answer]
Self ACE interaction -> database -> delete

**Reply by SOF-Ice** [reply]
thanks for your help
